packages/angular-cli/lib/config/schema.json

```diff
@@ -293,6 +293,9 @@
       },
       "additionalProperties": false
     },
+    "packageManager": {
+        "enum": [ "npm", "cnpm", "yarn" ]
```

Comment:
There should be another value here, `"default"`, which enables the check. People might want to use `npm` without having the warning shown up.